napari_animation/_qt/savedialog_widget.py

```diff
@@ -0,0 +1,98 @@
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import (
+    QCheckBox,
+    QComboBox,
+    QDoubleSpinBox,
+    QFileDialog,
+    QHBoxLayout,
+    QLabel,
+    QSpinBox,
+    QWidget,
+)
+
+
+class SaveDialogWidget(QFileDialog):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def getSaveFileName(
```

Comment:
this method now gets more than just the filename, should maybe be called `getAnimationParameters`